todo_app/app.py

```diff
@@ -9,11 +9,16 @@
 
 @app.route('/')
 def index():
-    items = get_items()
+    items = get_trello_items()
```

Comment:
Nice, the fact these are all pretty minimal changes in this file is a good sign you've stuck effectively to the existing interface 👍